### ISSUE TITLE
run_mpc.py: quadcopter just flew away

### ISSUE THREAD

**Issue by t-thanh** [uzh-rpg/high_mpc#1]
Hi,

I just clone the repository and test with `python run_mpc.py` after deleting some comments to record the animation. However the drone seems to be out of control - just fly away from the pendulum - see the attached video
https://user-images.githubusercontent.com/1252322/107228130-9668a100-6a1c-11eb-8d4f-89a10dbf02d3.mp4

Is that something missing from the set-up?

Best regards,

**Comment by yun-long** [comment]
Hi,

I have never encountered such a problem before. Are you able to run the code (without deleting anything) when you clone the code.

**Comment by t-thanh** [comment]
Hi @yun-long ,

Yes, I'm able to run the code like it is from the repository.
output of `python run_mpc.py` is:

```
2021-02-17 23:54:06.241345: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcudart.so.11.0
./mpc/saved/mpc_v1.so
No handles with labels found to put in legend.
No handles with labels found to put in legend.
No handles with labels found to put in legend.
No handles with labels found to put in legend.
0.011666536331176758
0.005145549774169922
0.0045680999755859375
0.004530191421508789
0.004487037658691406
0.004718303680419922
0.0045926570892333984
....
```

**Comment by t-thanh** [comment]
Fix by downgrade the matplotlib from 3.3.3 (python3.8) to 3.1.1
